neutrinordp/xrdp-neutrinordp.c

```diff
@@ -182,19 +182,33 @@ lxrdp_connect(struct mod *mod)
         }
 
 #endif
-        LOG(LOG_LEVEL_ERROR, "freerdp_connect Failed to "
-            "destination :%s:%d",
+	mod->pid = g_getpid();
+        LOG(LOG_LEVEL_ERROR, "NeutrinoRDP: connection [Failed]," 
+            " destination [%s:%d], username [%s], source [%s:%s],"
+	    " pamusername [%s], pid [%d]",
             mod->inst->settings->hostname,
-            mod->inst->settings->port);
+            mod->inst->settings->port,
+	    mod->inst->settings->username,
+	    mod->client_info.client_addr,
+	    mod->client_info.client_port,
+	    mod->pamusername,
+	    mod->pid);
         return 1;
     }
     else
     {
-        LOG(LOG_LEVEL_INFO, "freerdp_connect returned Success to "
-            "destination :%s:%d",
+	mod->pid = g_getpid();
```

Comment:
Suggest removing this line and replacing `mod->pid` in the logging call with `g_getpid()`.

Reply:
I'll follow your suggestion.

Reply:
Modified by f33a381.